### DISCUSSION TITLE
[Feature request] Hotkey output toggle

### DISCUSSION THREAD

**Discussion by healthazard** [File-New-Project/EarTrumpet#1012] [Ideas]
I like inclusion of shortcuts/hotkeys but a hotkey to toggle (change) outputs would be a killer feature.

PS: also ability to add WIN key to hotkey combo would be nice.

Thanks, great app.

**Comment by riverar** [comment]
@healthazard We have an add-on in the dev builds called EarTrumpet Actions that has all sorts of programmable features/hotkeys you might be interested in. Check it out! (Link available on https://eartrumpet.app)

**Reply by healthazard** [reply]
Sorry, it took me a while to check responses here.

Dev build looks nice, now I just have to check if I can make one hotkey a toggle button when I press it - it goes down the Audio output list one by one, one press one move on the output list.
Thanks :D

**Reply by healthazard** [reply]
> @healthazard We have an add-on in the dev builds called EarTrumpet Actions that has all sorts of programmable features/hotkeys you might be interested in. Check it out! (Link available on https://eartrumpet.app)

I've been using dev build for a week or two, now. I like that I can go to exact output via hotkey however I would still love to see the toggle hotkey as well, because I have mutliple outputs that sometime use different output devices that are sometime plugged in, other times not. So toggling through what is currently available via hotkey would be grand. I know I'm nitpicking, love the app tho, keep it up :D
